### DISCUSSION TITLE
Could I ask questions about GraphicsMagick?

### DISCUSSION THREAD

**Discussion by Lee52** [ImageMagick/ImageMagick#5428] [Help]
I see GraphicsMagick use sourceforge, but I guess GM and IM still have something in common?
https://sourceforge.net/p/graphicsmagick/bugs/

**Comment by snibgo** [comment]
The current GM and IM have a common ancestor, 20 years ago, which is a very long time. I have never used GM, and can't answer questions about it. If you have questions about GM, I suggest you ask on the forums you link to.

**Reply by Lee52** [reply]
thank you, it makes sense
